src/altinn_model_publisher/mapper/mapper_utils.py

```diff
@@ -0,0 +1,177 @@
+"""Utility methods for mapping process."""
+import os
+import re
+from typing import Dict, Optional
+
+from datacatalogtordf import Agent
+from modelldcatnotordf.modelldcatno import SimpleType
+from xmlschema import XMLSchema
+
+from altinn_model_publisher.organizations.organizations import map_shortname_to_org
+
+ORG_URI = os.getenv(
+    "ORGANIZATION_CATALOGUE_URI",
+    "https://organization-catalogue.fellesdatakatalog.digdir.no",
+)
+SELF_URI = os.getenv(
+    "ALTINN_MODEL_PUBLISHER_URI", "https://altinn-model-publisher.digdir.no"
+)
```

Comment:
Hadde vært fint å samle alle miljøvariablene på ett sted. Ellers blir de vanskeligere å spore jo flere av de vi har.

Reply:
Er nok rot her i denne, så setter heller opp en egen pr for det.